single-node/user-data

```diff
@@ -5,7 +5,10 @@ set -e
 export ETCD_ENDPOINTS="http://127.0.0.1:2379"
 
 # Specify the version (vX.Y.Z) of Kubernetes assets to deploy
-export K8S_VER=v1.2.2_coreos.0
+export K8S_VER=v1.2.3_coreos.0-cni
+
+# Hyperkube image repository to use.
+export HYPERKUBE_IMAGE_REPO=quay.io/calico/hyperkube
```

Comment:
@aaronlevy how would this feature flag work? I'd like to remove the choice between hyperkubes with CNI vs those without. Would this allow us to ship one container image?